.github/workflows/renovate.yml

```diff
@@ -27,8 +25,14 @@ jobs:
           app-id: ${{ vars.BOT_APP_ID }}
           private-key: ${{ secrets.BOT_PRIVATE_KEY }}
       - env:
-          RENOVATE_TOKEN: ${{ secrets.RENOVATE_TOKEN || steps.generate-token.outputs.token }}
+          RENOVATE_TOKEN: ${{ steps.generate-token.outputs.token || secrets.RENOVATE_TOKEN }}
+          RENOVATE_GIT_AUTHOR: ${{ vars.RENOVATE_GIT_AUTHOR }}
         run: |
+          if [ -n "${{ steps.generate-token.outputs.token }}" ] && [ -n "$RENOVATE_GIT_AUTHOR" ]; then
+            echo "# :warning: RENOVATE_GIT_AUTHOR is set when using GitHub App token" >> $GITHUB_STEP_SUMMARY
```

Comment:
When `RENOVATE_GIT_AUTHOR` is not specified, it needs an extra request to get the `id` of the user. So IMO, it is actually recommended to set the `RENOVATE_GIT_AUTHOR` explicitly to avoid the extra request. I suppose the original logic might be warn the users when NO `RENOVATE_GIT_AUTHOR` is specified and it falls back to the default value `Renovate GitHub Bot <github@renovatebot.com>`, right?

[1] https://docs.renovatebot.com/modules/platform/github/#running-as-a-github-app

Reply:
I consider when users need to set RENOVATE_GIT_AUTHOR, they should be responsible for the correctness. So I don't issue the default of RENOVATE_GIT_AUTHOR.

Reply:
It makes sense to some extent, but the behavior would be a warning when the user try to save a redundant API request. Actually, I would like to recommend users to set the `RENOVATE_GIT_AUTHOR` explicity and how to set it correctly in the documentation, so this is why I have some concern here.

Reply:
it does need to document it.

Reply:
@huxuan I have added note in the section of Renovate configuration, please see the review from rtd.